test/a11y.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+const async = require('async');
+const pa11y = require('pa11y');
+const reporter = require('pa11y/reporter/cli');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).